services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    order = Order.objects.create(
+        user=(get_user_model().objects.get(username=username))
```

Comment:
The 'create_order' function should handle the case where a user with the given username does not exist. Consider using 'get_object_or_404' or handling the 'DoesNotExist' exception to provide a more user-friendly error message.